coala_quickstart/Constants.py

```diff
@@ -10,6 +10,27 @@
     "Java": {"JavaPMD", "CheckstyleBear"},
     "Python": {"PycodestyleBear"}}
 
+ALL_CAPABILITIES = {
+    'Code Simplification',
+    'Commented Code',
+    'Complexity',
+    'Documentation',
+    'Duplication',
+    'Formatting',
+    'Grammar',
+    'Memory Leak',
+    'Missing Import',
+    'Redundancy',
+    'Security',
+    'Smell',
+    'Spelling',
+    'Syntax',
+    'Undefined Element',
+    'Unreachable Code',
+    'Unused Code',
+    'Variable Misuse'
+}
+
```

Comment:
yeah, but it's still probably a good idea to generate it so that we don't have to maintain it

But I agree that this is pretty static; create an issue for this (difficulty/low) if someone else wants to do it